content/goldcard-holders-faq/tax.md

```diff
@@ -63,3 +63,18 @@ The National Taxation Bureau provides free professional taxation consultation se
 During "tax season" there are dedicated English language counters for tax filing at
 National Taxation Bureau offices in major cities.
 
+## My salary is paid by an overseas company, where do I pay tax?
+Taiwan, and sometimes also in the overseas country.
+
+If you stay in Taiwan for more than 90 days in a year, money you get for work you perform while resident in Taiwan
+ is considered Taiwan-sourced income. This is true regardless of whether your salary is paid to you in an account
```

Comment:
What about the 183 days thing? Should that be included here?

Reply:
Yes, I agree there is a place for the 183 days thing in this section. It probably deserves an FAQ entry of its own.